app/controllers/case_logs_controller.rb

```diff
@@ -9,21 +9,18 @@ class CaseLogsController < ApplicationController
   before_action :find_resource, except: %i[create index edit]
 
   def index
-    set_session_filters
-
-    all_logs = current_user.case_logs
-    unpaginated_filtered_logs = filtered_case_logs(filtered_collection(all_logs, search_term))
+    filters = set_updated_session_filters
 
     respond_to do |format|
       format.html do
+        all_logs = current_user.case_logs
+
+        unpaginated_filtered_logs = filtered_case_logs(all_logs, search_term, filters)
+
         @pagy, @case_logs = pagy(unpaginated_filtered_logs)
         @searched = search_term.presence
         @total_count = all_logs.size
       end
-
-      format.csv do
-        send_data unpaginated_filtered_logs.to_csv, filename: "logs-#{Time.zone.now}.csv"
```

Comment:
i.e. we could just call `email_csv` from in here as before?

Reply:
I don't think that's quite right as we're not returning that data anymore. The action is to send an email vs return logs.